src/index.js

```diff
@@ -190,8 +195,19 @@ export async function crawlFrameworkPkgs(options) {
 }
 
 /** @type {import('./index.d.ts').findDepPkgJsonPath} */
-export async function findDepPkgJsonPath(dep, parent) {
+export async function findDepPkgJsonPath(dep, parent, workspaceRoot) {
   if (pnp) {
+    // if we're in a workspace and the dep is a workspace dep, 
+    // then we'll try to resolve to it's real location
+    if (workspaceRoot) {
+      const locator = pnpWorkspaceLocators.find((root) => root.name === dep)
+      if (locator) {
+        // relative to the workspace
+        const [, relativePath] = locator.reference.split('workspace:')
```

Comment:
according to https://yarnpkg.com/advanced/pnpapi#packagelocator  the reference value is "whatever the pnp api implementation puts there" so uuuhh, i don't think we can rely on that